### PR TITLE
📚 DOCS: Fix deprecated import

### PR DESCRIPTION
**Pull request by firasm**
Hello,

I was playing with this package and noticed that there are a couple of warning messages with the current important commands. 

Also one other suggestion is to add some code for people to export the html to a file if they happen to not be working in an interactive python session. Happy to remove this if it doesn't make sense for others.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,11 @@ A footnote [^1]
 """)
 tokens = md.parse(text)
 html_text = md.render(text)
+
+## To export the html to a file, uncomment the lines below:
+#text_file = open("output.html", "w")
+#text_file.write(html_text)
+#text_file.close()
```
Comment (chrisjsewell) [inline README.md:75]: This should either use:

```python
with open("output.html", "w") as handle:
   handle.write(html_text)
```

or probably better these days

```python
from pathlib import Path
Path("output.html").write_text(html_text)
```

**Review by chrisjsewell — CHANGES_REQUESTED** [review]
thanks @firasm, can you change the file writing code though

File: README.md (hunk 1 above)
Comment (chrisjsewell) [inline README.md:75]:
```suggestion
## To export the html to a file, uncomment the lines below:
# from pathlib import Path
# Path("output.html").write_text(html_text)
```

Comment (chrisjsewell) [reply README.md]: actually I'll just change it

**Comment by firasm** [comment]
Thanks! By the time I woke up today your suggested change was already done :-) 

Hard to unlearn how to write to files!

**Comment by chrisjsewell** [comment]
Yeh I was too impatient lol

**Pull request merged**